tests/parameter_handler/parameter_handler_3_with_space.output

```diff
@@ -0,0 +1,15 @@
+
+# Listing of Parameters
+# ---------------------
+subsection Testing
+  # docs 3
+  set double      = 3.1415926
+  set int         = 3         # default: 1
+
+  # docs 1
+  set string list1 = a, b, c   # default: a
+  #docs 2  
```

Comment:
does the test pass? There is white space in `docs 1` and no whitespace here. I wonder why.